ci-operator/step-registry/ipi/conf/gcp/osdisk/disksize/ipi-conf-gcp-osdisk-disksize-commands.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ -z "${COMPUTE_DISK_SIZEGB}" ] || [ -z "${CONTROL_PLANE_DISK_SIZEGB}" ]; then
```

Comment:
here should be use `&&` instead of `||`, right?

Reply:
So you suggest to move on if any of them non-empty?

Reply:
right